spec/support/response_json.rb

```diff
@@ -4,3 +4,6 @@ def response_json
   end
 end
 
+RSpec.configure do |config|
+  config.include ResponseJSON
+end
```

Comment:
Why does this show here, when in the code it is included in the rails_helper instead of response_json?